managed_vms/endpoints/pom.xml

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
nit. We've been adding license headers to new pom.xml files (has to appear after `<?xml ...>` line.

Also, the XML attributes to the `<project>` tag are optional, so we've been leaving them out.